.rtd-environment.yml

```diff
@@ -61,4 +61,4 @@ dependencies:
 - glymur
 - pip:
     - sunpy-sphinx-theme
-    - git+https://github.com/sphinx-gallery/sphinx-gallery
```

Comment:
sphinx-gallery and sphinx-astropy do two different things we are going to need both :smile: